app/Modules/Bookings/RatingRequirementNotMetException.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Modules\Bookings;
+
+use Exception;
+use Nuwave\Lighthouse\Exceptions\RendersErrorsExtensions;
+
+class RatingRequirementNotMetException extends Exception implements RendersErrorsExtensions
+{
+    protected $message = 'Your rating is not high enough to book that position.';
+
+    /**
+     * Returns true when exception message is safe to be displayed to a client.
+     *
+     * @return bool
+     *
+     * @api
+     */
+    public function isClientSafe()
+    {
+        return true;
+    }
+
+    /**
+     * Returns string describing a category of the error.
+     *
+     * Value "graphql" is reserved for errors produced by query parsing or validation, do not use it.
+     *
+     * @return string
+     *
+     * @api
+     */
+    public function getCategory()
+    {
+        return 'bookings-validation';
```

Comment:
Might it be best to stick to the validation category? Just thinking how we are going to handle this on client side

Reply:
The other exception (Overlapping) just has `bookings` - change both?

Reply:
sounds good to me